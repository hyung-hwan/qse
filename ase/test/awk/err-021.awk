BEGIN {
	xx = 20;
	split ("a  b c", xx);
}
