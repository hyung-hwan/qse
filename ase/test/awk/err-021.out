BEGIN {
	xx = 20;
	split ("a  b c",xx);
}

RUN ERROR: CODE [97] LINE [3] cannot change a scalar value to a map
